.eslintrc.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
maybe more consistent in yaml format, like with the travis config

Reply:
Guess you have a point. Reverted to `yml`.